src/renderer/components/MarketChart/MarketChart.vue

```diff
@@ -284,8 +291,16 @@ export default {
 }
 </script>
 
-<style scoped>
+<style lang="postcss" scoped>
 .MarketChart {
   min-height: 315px
 }
+
+.MarketChart__Loader__Container {
+  @apply .flex;
```

Comment:
Here you can use `.items-center` and `.justify-center`

Reply:
Darn, thanks for that!